back/lib/tasks/single_use/20250326_migrate_forms_separate_title_description_pages.rake

```diff
@@ -0,0 +1,284 @@
+namespace :migrate_custom_forms do
+  desc 'Migrate forms to have separate pages for title and body.'
+  task :separate_title_body_pages, [] => [:environment] do
+    reporter = ScriptReporter.new
+    Tenant.safe_switch_each do |tenant|
+      {
+        'ideation_page1' => 'title_page',
+        'ideation_page2' => 'uploads_page',
+        'ideation_page3' => 'details_page'
+      }.each do |code_from, code_to|
+        CustomField.where(input_type: 'page', code: code_from).update_all(code: code_to)
+      end
+
+      CustomForm.all.each do |form|
+        # If we have a real title page with first the body field then the title field,
+        # we need to move the title page after the body field (in this case, before the
+        # title field), to avoid creating an extra page in front of the body page.
+        # fields_per_page, title_page, body_page = rake_20250326_group_field_by_page(form.custom_fields)
+        # if title_page == body_page && title_page.code == 'title_page'
+        #   title_field = fields_per_page[title_page].find { |field| field.code == 'title_multiloc' }
+        #   body_field = fields_per_page[title_page].find { |field| field.code == 'body_multiloc' }
+        #   if title_field && body_field && title_field.ordering > body_field.ordering
+        #     # We can just move the title page if it's right before the body field,
+        #     # because we will add a body page in front of the body field later.
+        #     prev_ordering = rake_20250326_reorder_and_report_page(title_page, title_field.ordering, reporter)
+        #     if title_page.ordering != (body_field.ordering - 1)
+        #       rake_20250326_create_and_report_page(rake_20250326_new_normal_page(form), prev_ordering, reporter, 'add_body_page')
+        #     end
+        #   end
+        # end
```

Comment:
Is this commented out for a reason?

Reply:
Yes, I will delete it. I commented it out because it seemed unnecessary, and indeed it was. But I wasn't 100% sure, so I didn't want to throw away the code yet.